.github/workflows/native-tests.yml

```diff
@@ -26,6 +26,15 @@ jobs:
         if: runner.os == 'Linux'
       - run: .github/scripts/maybe-with-graalvm-home.sh nativeTests
         shell: bash
+      - name: Copy artifacts
+        run: .github/scripts/maybe-with-graalvm-home.sh copyLauncher artifacts/
```

Comment:
That line fails on the Windows CI with a `FileAlreadyExistsException`. I don’t understand what is going on because what I do is similar to what is done in the `ci.yml`. Does `nativeTests` also copy the JVM?

Reply:
Cherry-picking [that commit](https://github.com/alexarchambault/coursier/commit/d20552adb2eeaad910d5bb22ec68ef5777927ece) should fix that (seems I can't push on the PR branch here).